src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -264,14 +264,25 @@
 (defn describe-table-fks
   "Default implementation of [[metabase.driver/describe-table-fks]] for SQL JDBC drivers. Uses JDBC DatabaseMetaData."
   [driver db-or-id-or-spec-or-conn table & [db-name-or-nil]]
-  (if (instance? Connection db-or-id-or-spec-or-conn)
-    (describe-table-fks* driver db-or-id-or-spec-or-conn table db-name-or-nil)
-    (sql-jdbc.execute/do-with-connection-with-options
-     driver
-     db-or-id-or-spec-or-conn
-     nil
-     (fn [^Connection conn]
-       (describe-table-fks* driver conn table db-name-or-nil)))))
+  (sql-jdbc.execute/do-with-connection-with-options
+   driver
+   db-or-id-or-spec-or-conn
+   nil
+   (fn [^Connection conn]
+     (describe-table-fks* driver conn table db-name-or-nil))))
+
+(defmulti describe-fks-sql
```

Comment:
You should mention in this ns's docstraing that `describe-table-fks` is deprecated and we should implement `describe-fks-sql` instead.